### ISSUE TITLE
Bug: onedrive is started twice in the setup with SDDM

### ISSUE THREAD

**Issue by iamkarlson** [abraunegg/onedrive#2406]
### Describe the bug

Onedrive process is started twice in Arch.

### Operating System Details

```shell
❯ uname -a
Linux beryl 6.3.4-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 24 May 2023 17:44:00 +0000 x86_64 GNU/Linux

❯ lsb_release -a
LSB Version:    n/a
Distributor ID: Arch
Description:    Arch Linux
Release:        rolling
Codename:       n/a
```

### Client Installation Method

From Distribution Package

### OneDrive Account Type

Personal

### What is your OneDrive Application Version

onedrive v2.4.23

### What is your OneDrive Application Configuration

```shell
❯ onedrive --display-config
Configuration file successfully loaded
onedrive version                             = v2.4.23
Config path                                  = /home/iamkarlson/.config/onedrive
Config file found in config path             = true
Config option 'sync_dir'                     = /home/iamkarlson/onedrive
Config option 'enable_logging'               = false
Config option 'log_dir'                      = /var/log/onedrive/
Config option 'disable_notifications'        = false
Config option 'min_notify_changes'           = 5
Config option 'skip_dir'                     =
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ~*|.~*|*.tmp
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 300
Config option 'monitor_log_frequency'        = 6
Config option 'monitor_fullscan_frequency'   = 12
Config option 'read_only_auth_scope'         = false
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'cleanup_local_files'          = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'space_reservation'            = 52428800
Config option 'application_id'               =
Config option 'azure_ad_endpoint'            =
Config option 'azure_tenant_id'              = common
Config option 'user_agent'                   =
Config option 'force_http_11'                = false
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 10485760
Config option 'operation_timeout'            = 3600
Config option 'sync_root_files'              = true
Selective sync 'sync_list' configured        = true
sync_list contents:
/Passwords
/Apps
/important
/books
/Documents
/fun
/Study
/Images
/Docs-IDs-Passports
Config option 'sync_business_shared_folders' = false
Business Shared Folders configured           = false
Config option 'webhook_enabled'              = false
```

### What is your 'curl' version

```shell
❯ curl --version
curl 8.1.1 (x86_64-pc-linux-gnu) libcurl/8.1.1 OpenSSL/3.0.8 zlib/1.2.13 brotli/1.0.9 zstd/1.5.5 libidn2/2.3.4 libpsl/0.21.2 (+libidn2/2.3.4) libssh2/1.10.0 nghttp2/1.53.0
Release-Date: 2023-05-23
Protocols: dict file ftp ftps gopher gophers http https imap imaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL threadsafe TLS-SRP UnixSockets zstd
```

### Where is your 'sync_dir' located

Local

### What are all your system 'mount points'

```shell
N/A
```

### What are all your local file system partition types

```shell
N/A
```

### How do you use 'onedrive'

As prescribed in config doc.

### Steps to reproduce the behaviour

I configured onedrive via systemd. However, it's quite often that some *other* process is running the sync forked from sddm process. When this happens, I usually see the following picture:
```
❯ journalctl --user-unit=onedrive -f             
Jun 01 11:21:44 beryl systemd[1634]: onedrive.service: Scheduled restart job, restart counter is at 3235.
Jun 01 11:21:44 beryl systemd[1634]: Stopped OneDrive Free Client.
Jun 01 11:21:44 beryl systemd[1634]: Started OneDrive Free Client.
Jun 01 11:21:44 beryl (onedrive)[125208]: onedrive.service: ProtectHostname=yes is configured, but UTS namespace setup is prohibited (container manager?), ignoring namespace setup.
Jun 01 11:21:44 beryl onedrive[125208]: Configuration file successfully loaded
Jun 01 11:21:44 beryl onedrive[125208]: Configuring Global Azure AD Endpoints
Jun 01 11:21:44 beryl onedrive[125208]: User Configured Rate Limit: 10485760
Jun 01 11:21:44 beryl onedrive[125208]: ERROR: onedrive application is already running - check system process list for active application instances
Jun 01 11:21:44 beryl systemd[1634]: onedrive.service: Main process exited, code=exited, status=1/FAILURE
Jun 01 11:21:44 beryl systemd[1634]: onedrive.service: Failed with result 'exit-code'.
Jun 01 11:21:47 beryl systemd[1634]: onedrive.service: Scheduled restart job, restart counter is at 3236.
Jun 01 11:21:47 beryl systemd[1634]: Stopped OneDrive Free Client.
Jun 01 11:21:47 beryl systemd[1634]: Started OneDrive Free Client.
Jun 01 11:21:47 beryl (onedrive)[125261]: onedrive.service: ProtectHostname=yes is configured, but UTS namespace setup is prohibited (container manager?), ignoring namespace setup.
Jun 01 11:21:47 beryl onedrive[125261]: Configuration file successfully loaded
Jun 01 11:21:48 beryl onedrive[125261]: Configuring Global Azure AD Endpoints
Jun 01 11:21:48 beryl onedrive[125261]: User Configured Rate Limit: 10485760
Jun 01 11:21:48 beryl onedrive[125261]: ERROR: onedrive application is already running - check system process list for active application instances
Jun 01 11:21:48 beryl systemd[1634]: onedrive.service: Main process exited, code=exited, status=1/FAILURE
Jun 01 11:21:48 beryl systemd[1634]: onedrive.service: Failed with result 'exit-code'.
Jun 01 11:21:51 beryl systemd[1634]: onedrive.service: Scheduled restart job, restart counter is at 3237.
Jun 01 11:21:51 beryl systemd[1634]: Stopped OneDrive Free Client.

❯ systemctl --user status onedrive.service
● onedrive.service - OneDrive Free Client
     Loaded: loaded (/usr/lib/systemd/user/onedrive.service; enabled; preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Thu 2023-06-01 11:22:04 CEST; 1s ago
       Docs: https://github.com/abraunegg/onedrive
    Process: 125445 ExecStart=/usr/bin/onedrive --monitor (code=exited, status=1/FAILURE)
   Main PID: 125445 (code=exited, status=1/FAILURE)
        CPU: 76ms
```

I didn't even know that sddm can start any process besides the main one. Is there any automatic registration of `onedrive` somewhere in the system? how does it work in general?

### Complete Verbose Log Output

```shell
N/A
```

**Comment by abraunegg** [comment]
@iamkarlson 
> onedrive is started twice in the setup with SDDM

Simple Desktop Display Manager (SDDM) must be looking at your enabled systemd services - and must not be seeing that the service is already started, before starting it again.

This is not a 'onedrive' problem - please take your issue to Arch Linux Forums.

> Is there any automatic registration of onedrive somewhere in the system?

No there is not. You must manually enable systemd services.

> how does it work in general?

Please refer to the FAQ: https://github.com/abraunegg/onedrive/wiki/Frequently-Asked-Questions

Closing issue - as the client is doing the correct thing here - Arch and/or SDDM is what is broken.

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
